### PR TITLE
Wrap instances of session.get in the _get_response method

### PR DESCRIPTION
**Pull request by reginafcompton**
@evz - can you take a look at this? I integrated `_get_response` throughout import_data: it should help minimize errors having to do with 504 timeouts in ocd.datamade.us.

### CODE REVIEW

File: councilmatic_core/management/commands/import_data.py [hunk 1]
```diff
@@ -449,32 +456,33 @@ def grab_people(self):
...
-
-        return page_json
+        r = self._get_response(url)
+
+        if r:
+            page_json = json.loads(r.text)
+
+            # save image to disk
+            if page_json['image']:
+                r = session.get(page_json['image'], verify=False)
```
Comment (evz) [inline councilmatic_core/management/commands/import_data.py]: How come this isn't using the `_get_response` thing?

Comment (reginafcompton) [reply councilmatic_core/management/commands/import_data.py]: Ah, I overlooked that one....I'll need to account for the `verify=False` argument.

**Comment by reginafcompton** [comment]
@evz - I added `*kwargs` to `_get_response` (à la the [requests library](https://github.com/requests/requests/blob/2872f17ab86583294907af84b9c36e63cd9a91ad/requests/api.py#L61)).

**Comment by reginafcompton** [comment]
I am closing this PR, since we do not want to wrap all GET requests in `_get_response`, which throws a warning, rather than raising an error. It makes sense when retrieving the shape_url, since its fine to continue without that info. Otherwise, we want `import_data` to raise an error.

**Pull request closed without merging**